test/sanity/secrets.yaml

```diff
@@ -0,0 +1,3 @@
+NodeStageVolumeSecret:
```

Comment:
what is this file for?

Reply:
this is the format to pass smb server secrets